internal/cmd/argus/scrape-configs/delete/delete.go

```diff
@@ -0,0 +1,126 @@
+package delete
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/client"
+	argusUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/argus/utils"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/spinner"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus"
+	"github.com/stackitcloud/stackit-sdk-go/services/argus/wait"
+)
+
+const (
+	jobNameArg = "JOB_NAME"
+
+	instanceIdFlag = "instance-id"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	JobName    string
+	InstanceId string
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   fmt.Sprintf("delete %s", jobNameArg),
+		Short: "Deletes a Scrape Config job from an Argus instance",
```

Comment:
This is valid for all scrape config commands: I think it's not needed to start with capital letter and we should have the full word "configuration". Also, not sure if we really need to specify job because also in the Portal it is just identified as scrape configuration.

So I would suggest that we just have "Deletes a scrape configuration from an Argus instance"